src/dev/code_coverage/shell_scripts/generate_team_assignments_and_ingest_coverage.sh

```diff
@@ -32,13 +32,16 @@ TEAM_ASSIGN_PATH=$5
 # Build team assignments dat file
 node scripts/generate_team_assignments.js --verbose --src .github/CODEOWNERS --dest $TEAM_ASSIGN_PATH
 
-for x in jest functional; do
-  echo "### Ingesting coverage for ${x}"
+echo "### Ingesting coverage for functional"
 
-  COVERAGE_SUMMARY_FILE=target/kibana-coverage/${x}-combined/coverage-summary.json
+COVERAGE_SUMMARY_FILE=target/kibana-coverage/functional-combined/coverage-summary.json
+node scripts/ingest_coverage.js --verbose --path ${COVERAGE_SUMMARY_FILE} --vcsInfoPath ./VCS_INFO.txt --teamAssignmentsPath $TEAM_ASSIGN_PATH
 
-  node scripts/ingest_coverage.js --verbose --path ${COVERAGE_SUMMARY_FILE} --vcsInfoPath ./VCS_INFO.txt --teamAssignmentsPath $TEAM_ASSIGN_PATH
-done
+echo "### Ingesting coverage for jest"
+# Need to override COVERAGE_INGESTION_KIBANA_ROOT since json file has original intake worker path
+COVERAGE_SUMMARY_FILE=target/kibana-coverage/jest-combined/coverage-summary.json
+export COVERAGE_INGESTION_KIBANA_ROOT=/dev/shm/workspace/kibana
+node scripts/ingest_coverage.js --verbose --path ${COVERAGE_SUMMARY_FILE} --vcsInfoPath ./VCS_INFO.txt --teamAssignmentsPath $TEAM_ASSIGN_PATH
```

Comment:
Why duplicate instead of loop?

Reply:
Line #43 is specific for jest, I thought about loop + if but then decided to do it this way

Reply:
@wayneseymour do you want me to change it with for+if ?

Reply:
Nah it's all good.  I just keep a lookout for duplication.